privacyidea/static/components/token/controllers/tokenControllers.js

```diff
@@ -46,7 +46,7 @@ myApp.controller("tokenController", function (TokenFactory, ConfigFactory,
             $scope.params.userid = "*" + ($scope.userIdFilter || "") + "*";
             $scope.params.resolver = "*" + ($scope.resolverFilter || "") + "*";
             $scope.params.pagesize = $scope.token_page_size;
-            $scope.params.sortBy = $scope.sortBy;
```

Comment:
What about the ``sortBy`` in
https://github.com/privacyidea/privacyidea/blob/5f6aa31a467ebf01cd25efad397c15ebf4b4c982/privacyidea/static/components/token/controllers/tokenChallengesController.js#L20

Reply:
This is the endpoint GET /token/challenges

This should also be "sortby" lowercase.
Will do.

Reply:
Fixed it in 15244ca8
although the token challenge list is probably not that long.
Thanks for noting it!